README.md

```diff
@@ -2,29 +2,85 @@
 
 <h1 align="center">Tropy Archive Plugin</h1>
 
-Tropy plugin for exporting items into a single zip archive. This includes all the metadata, as well as the photo files.
+Tropy plugin for exporting items into a single zip archive.
+This includes all the metadata, as well as the photo files.
 
 ## Installation
 
-Download the appropriate [release](https://github.com/tropy/tropy-archive/releases) for your platform and install via the _Plugins_ preferences pane.
+- Download the `.zip` file from the [latest release](https://github.com/tropy/tropy-plugin-archive/releases.latest) on GitHub.
+- In Tropy, navigate to _Preferences… > Plugins_
+  and click _Install Plugin_ to select the downloaded ZIP file.
 
-Alternaively, navigate to the Tropy `plugins` folder (_Main menu -> Help -> Show plugins folder_) in the terminal and run
+## Configuration
 
-    npm install tropy-archive
+To configure the plugin, click its _Settings_ button in _Preferences > Plugins_:
 
-or
+In the plugin preferences,
 
-    git clone https://github.com/tropy/tropy-archive
+- the `Name` parameter names the particular _instance_ of the plugin,
+to make it easier to select that configuration from the Export menu.
+- the `Root folder` parameter specifies the name of the folder
+(inside the zip archive) where all the exported items will be placed.
+- the `Image folder` parameter specifies the name of the subdirectory within the Root folder where image files will be saved.
+By default this is `.`,
+which means the Root directory itself rather than a subdirectory
+- the `Save as` parameter specifies the default file name and location for the zip archive to be saved.
+- `Always ask?` can be checked to make the file picker dialog always be shown,
+even when the `Save as` parameter is set to specify a default location.
 
-## Configuration
+## Usage
+
+To trigger the plugin, select the entry in the export sub-menu in Tropy.
+
+<details>
```

Comment:
I'd rather leave out the information about developing and debugging here and instead link to our new plugin docs or the sample plugin. The reason is that if we have it here (and potentially other plugins) we have the information spread across many repos and we'll eventually fail to keep it up to date in all places.